### ISSUE TITLE
Form generation process has a lot of steps

### ISSUE THREAD

**Issue by estellecomment** [medic/medic-webapp#2115]
Forms are design in excel/google sheets, and then transformed into JSON and/or XML and uploaded to the webapp.
0.4 mostly has JSON forms. XML forms are sometimes also used alongside, for Medic Collect.
Note that one project can have many forms.

For JSON forms, the process right now is, from google sheets :
- For each form individually, use app script to generate JSON and send it to your email
- create a JSON file with an array, and copy-paste each emailed JSON form into the array.
- upload the JSON into webapp in [`Forms` tab](https://demo.app.medicmobile.org/medic/_design/medic/_rewrite/settings#forms) in settings.

For XML forms, the process is, from google sheets :
- For each form individually, download to xlsx format.
- run python script locally to generate xml (this gives strange errors if your python version is wrong)
- run upload script to push xml to webapp

Solutions will be discussed soon, stay tuned

**Comment by bishwas-medic** [comment]
One more issue with the conversion script for converting XLS Form to JSON:

- It doesn't include the choices for select questions
- It doesn't include most of the constraints, skip logic and validations

**Comment by estellecomment** [comment]
Priority for @bishwas-medic 's comment : 
choices : 5/5
range : 3/5

**Comment by nice-snek** [comment]
hi frand @mandric, U is coot

Please close or schedule before the end of this sprint. See [triaging old issues](https://github.com/medic/medic-docs/blob/master/md/dev/workflow.md#triaging-old-issues).

Nice Snek

**Comment by nice-snek** [comment]
hi frand @mandric, please [triage](https://github.com/medic/medic-docs/blob/master/development/workflow.md#triaging-old-issues) before the end of this sprint.

**Comment by mandric** [comment]
Important enough to leave in the backlog but not a priority right now?

**Comment by sglangevin** [comment]
This may not be relevant once medic-configurer is complete. Not sure what capabilities it will have with regard to JSON forms, though.

**Comment by garethbowen** [comment]
I think we can call this fixed. If there are more improvements to make they should be made to medic-configurer but most of this is solved.
